CIP-0067/registry.schema.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/cardano-foundation/CIPs/blob/master/CIP-?",
```

Comment:
https://cips.cardano.org/cips/cip67/registry.schema.json should (eventually) work.

e.g. https://cips.cardano.org/cips/cip10/registry.schema.json